code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -225,15 +225,15 @@
 
 /obj/machinery/portable_atmospherics/canister/halon
 	name = "Halon canister"
-	desc = "Halon, remove oxygen from high temperature fires and cool down the area"
+	desc = "Halon, removes oxygen from high temperature fires and cools down the area"
 	icon_state = "halon"
 	base_icon_state = "halon"
 	gas_type = /datum/gas/halon
 	filled = 1
 
 /obj/machinery/portable_atmospherics/canister/hexane
 	name = "Hexane canister"
-	desc = "hexane, highly flammable"
+	desc = "Hexane, highly flammable"
```

Comment:
This was a copy paste error, since you are here could you please change it to "Hexane, highly reactive"

Reply:
Alright